### ISSUE TITLE
Remove all sounds but the startup sound

### ISSUE THREAD

**Issue by yuriiz** [webtorrent/webtorrent-desktop#1067]
There is no way to turn off annoying sound notifications when interacting with application (adding torrent, removing torrent etc).

**What version of WebTorrent Desktop?** (See the 'About WebTorrent' menu)
Version 0.17.2 (0.97.2) (64-bit)

**What operating system and version?**
Xubuntu 16.04

**What did you do?**
Opened Edit->Preferences

**What did you expect to happen?**
Too see an option to turn off super-annoying blip-blips on every action.

**What actually happened?**
Didn't found it.

**Comment by yuriiz** [comment]
Workaround under linux is to mute "Chromium" application under PulseAudio Volume Control but this mutes media player too and it is becoming usable only with external media player.

**Comment by kaotisk-hund** [comment]
I started this [PR](https://github.com/feross/webtorrent-desktop/pull/1068) but I can't complete it.

**Comment by feross** [comment]
Thanks for the issue, but I don't want a preference for every trivial option that a user could want. I think we should just do the right thing, which is probably to remove the sounds entirely it seems.

**Comment by feross** [comment]
Re-opening to use this as the tracking issue.

**Comment by mifi** [comment]
I think the sounds are cute

**Comment by feross** [comment]
@mifi Me too, but they're pretty controversial

**Comment by rjmill** [comment]
@feross I agree with you on not having preferences for trivial things, but I think a checkbox for muting the sounds would be completely appropriate.

I'd toggle the sounds all day if I could. I don't like them most of the time, but they're really convenient when I am actively waiting for a torrent to download. The fact that the sounds are controversial and polarizing is a good sign that each user should be able to decide for themselves what to do, IMO.
